src/widgets/dropdown/widget-dropdown-model.js

```diff
@@ -0,0 +1,16 @@
+var cdb = require('cartodb.js');
+
+/**
+ * Default widget dropdown model
+ */
+module.exports = cdb.core.Model.extend({
+  defaults: {
+    collapsed: false,
+    flags: {
```

Comment:
I'm not sure, it was originally written this way. I didn't want to change the existing behaviour.